pkg/server/server.go

```diff
@@ -76,9 +76,9 @@ var immudbTextLogo = " _                               _ _     \n" +
 
 // Initialize initializes dependencies, set up multi database capabilities and stats
 func (s *ImmuServer) Initialize() error {
-	s.Logger.Infof("%s\n%s\n%s\n\n", immudbTextLogo, version.VersionStr(), s.Options)
```

Comment:
I don't understand this well, but can we create an issue to track this separately? Because I don't see it as a blocker for adding json log support, if it is, you can lemme know.

Reply:
ok based on our discussion, I've added some steps, please verify for the same